src/Adobe/composer.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "socialiteproviders/adobe",
```

Comment:
Can you please add this to the split script (monorepo builder file in repo root)